about.html

```diff
@@ -0,0 +1,166 @@
+                </div>
+            </div>
+        </header>
+    </section>
+
+    <section class="logo_sect">
+        <div class="main_about">
+            <h3 class="pro_head">Web Developer Conference 2022 logo</h3>
+            <hr class="about_hr">
+        </div>
+        <p class="logo_description">
+            The Logo of the Web Developer Conference 2022 was descided through the
+            logo competition from 8. june to 11. july
+        </p>
+        <div class="conference_img">
+            <img class="logo" src="./images/nav.png" alt="Web Developer Conference logo">
+        </div>
+    </section>
+
+    <section class="history">
+        <div class="main_history">
+            <h3 class="head_history">See the History</h3>
+            <hr class="history_hr">
+        </div>
+        <p class="history_description">
+            Have a look at the two last Web Developer Conference which took place at Empire Riverside Hotel
+        </p>
+        <div class="history_img-div ">
+            <div class="img_h overlay">
+                <img class="hist_img" src="./images/history_1.jpeg" alt="Web Developer Conference logo">
+                <p class="h_para"><span class="year_hist">2021</span>Web Developer Conference at Empire Riverside
+                    Hotel</p>
+                <img class="hist_img" src="./images/history_2.jpeg" alt="Web Developer Conference logo">
+                <p class="h_para"><span class="year_hist">2018</span>Web Developer Conference at Empire Riverside
+                    Hotel</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="about_ptn">
+        <div class="main_ptn">
+            <h3 class="head_ptn">Partners</h3>
+            <hr class="partners_hr">
+        </div>
+        <div class="image_p">
+            <div class="p1"><img class="ptn_img" src="./images/partner/p1.png" alt=""></div>
+            <div class="p1"><img class="ptn_img" src="./images/partner/p2.jpg" alt=""></div>
+            <div class="p1"><img class="ptn_img" src="./images/partner/p3.png" alt=""></div>
+            <div class="p1"><img class="ptn_img" src="./images/partner/p4.png" alt=""></div>
+        </div>
+    </section>
+    
+    
+    <section>
+        <footer class="footer">
+            <div class="logo_f">
+                <img class="f_img" src="./images/nav.png" alt="">
+            </div>
+            <div class="rights_reserved">
+                <p class="rights">web Developer Conference 2022,Rights Reserved</p>
+                <p class="p_rights"> I have read the data protection regulations and accept them. By registering, I agree
+                    that Ebner Media Group GmbH & Co. KG may inform me by email about in-house products, such as events,
+                    white papers or webinars. The Ebner Media Group will only use my data for this purpose. In particular,
+                    there is no disclosure to third parties</p>
+            </div>
+        </footer>
```

Comment:
- [ ] Your footer is good and very similar to the template. But on the desktop view, the footer background color is dark with white text. Kindly change the colors to match the template.
 - Your footer:
 - 
<img width="1030" alt="image" src="https://user-images.githubusercontent.com/61919617/204302243-39a9ea40-7b1e-48ef-a0f2-c9703e0e4936.png">

 - Template footer:
 - 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/61919617/204302340-e5317eb9-7edc-476e-b84b-85e311865f46.png">

Reply:
Issue resolved. Thank you